src/components/stateful-enroll/data/hooks/useStatefulEnroll.js

```diff
@@ -30,10 +30,11 @@ const useStatefulEnroll = ({
 }) => {
   const { data: enterpriseCustomer } = useEnterpriseCustomer();
   const [transaction, setTransaction] = useState();
-  const optimizelyHandler = useOptimizelyEnrollmentClickHandler(
-    contentKey,
+  // TODO: What is the href/enrollmentURL for a statefulEnroll?
+  const optimizelyHandler = useOptimizelyEnrollmentClickHandler({
```

Comment:
Great catch that args currently passed weren't correct (and haven't been for awhile!).

Regarding your TODO comment, I don't believe we want to pass `href` to `useOptimizelyEnrollmentClickHandler`. If we do, it will trigger global.location.assign(href)`, but we want to rely on the `navigate('complete');` in `UserEnrollmentForm`'s `handleFormSubmissionSuccess` to use client-side router navigatation to the Exec Ed enrollment confirmation route.

Reply:
Sounds good, looks like its already doing that, removing the TODO 👍🏽